### PR TITLE
fix: updated words in toast after sending withdrawal transaction

### PR DESCRIPTION
**Pull request by impelcrypto**
**Pull Request Summary**

* fix: updated words in toast after sending withdrawal transaction

**Check list**

- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

=After=
image

### CODE REVIEW

File: src/i18n/en-US/index.ts [hunk 1]
```diff
@@ -352,7 +352,7 @@ export default {
       requiredClaimFirst: 'Please claim your rewards before sending transaction',
       requiredClaimFirstCompounding:
         '{message} -Disable compounding, claim your rewards and then enable compounding again',
-      successfullyWithdrew: 'Balance is successfully withdrew',
+      successfullyWithdrawn: 'Balance is successfully withdrawn',
```
Comment (codespool) [inline src/i18n/en-US/index.ts:355]: Not a blocker, but maybe drop "is" for brevity, so "Balance successfully withdrawn"
```suggestion
      successfullyWithdrawn: 'Balance successfully withdrawn',
```

Comment (impelcrypto) [reply src/i18n/en-US/index.ts]: Thanks, fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/928/commits/9706be51148fa5561727a6ef0dcbee83a805d46e) commit.

**Review by codespool — APPROVED** [review]

**Comment by codespool** [comment]
LGTM 👍

**Review by bobo-k2 — APPROVED** [review]

**Review by gluneau — APPROVED** [review]
LGTM

**Pull request merged**